### PR TITLE
Integrations - Add Search

### PR DESCRIPTION
**Pull request by timmo001**
Adds search to the config flow picker. Makes it much quicker to find what you want now.

image

image

image

### CODE REVIEW

**Comment by balloob** [comment]
You're on fire!

File: src/dialogs/config-flow/step-flow-pick-handler.ts [hunk 1]
```diff
@@ -1,34 +1,74 @@
...
 
 @customElement("step-flow-pick-handler")
 class StepFlowPickHandler extends LitElement {
-  public hass!: HomeAssistant;
-  public handlers!: string[];
+  @property() public hass!: HomeAssistant;
+  @property() public handlers!: string[];
+  @property() private filter?: string;
+
+  private _getHandlers = memoizeOne((h: any[], filter?: string) => {
```
Comment (balloob) [inline src/dialogs/config-flow/step-flow-pick-handler.ts]:
```suggestion
  private _getHandlers = memoizeOne((h: string[], filter?: string) => {
```

File: src/dialogs/config-flow/step-flow-pick-handler.ts [hunk 2]
```diff
@@ -1,34 +1,74 @@
...
+    const handlers: HandlerObj[] = h.map((handler) => {
+      return {
+        name: this.hass.localize(`component.${handler}.config.title`),
+        slug: handler,
+      };
+    });
+
+    if (filter) {
+      const options: Fuse.FuseOptions<any> = {
```
Comment (balloob) [inline src/dialogs/config-flow/step-flow-pick-handler.ts]: Should this be

```suggestion
      const options: Fuse.FuseOptions<HandlerObj> = {
```

Comment (timmo001) [reply src/dialogs/config-flow/step-flow-pick-handler.ts]: Yup. Thanks!

**Pull request merged**